server/fleet/app.go

```diff
@@ -707,6 +711,27 @@ func (f *Features) ApplyDefaults() {
 	f.EnableHostUsers = true
 }
 
+func (f *Features) Clone() (interface{}, error) {
+	return f.Copy(), nil
+}
+
+func (f *Features) Copy() *Features {
```

Comment:
figured this would be better than modifying the behavior of the default clone function defined in `cached_mysql` as it's unclear if there's other functionality relying on the current behavior.